src/scripts/blocks/countdown-timer/DisplayComponent.js

```diff
@@ -0,0 +1,116 @@
+import classnames from 'classnames';
+
+const { __ } = wp.i18n;
+const { Component, Fragment } = wp.element;
+const {
+  InspectorControls, RichText,
+} = wp.editor;
+const {
+  PanelBody, Button, DateTimePicker, ColorPicker, SelectControl,
+} = wp.components;
+const { applyFilters } = wp.hooks;
+const { PostMediaSelector } = benenson.components;
+
+class DisplayComponent extends Component {
+  render() {
+    const { attributes, setAttributes } = this.props;
+    const {
+      date = new Date(),
+      title = '',
+      backgroundColor = '',
+      backgroundId = null,
+      backgroundUrl = '',
+      textColor = '',
+    } = attributes;
+
+    const colourOptions = applyFilters('benenson.block.blockCountdown.colorOptions', [{
+      label: __('White', 'benenson'),
+      value: '',
+    }, {
+      label: __('Black', 'benenson'),
+      value: 'black',
+    }]);
+
+    const styles = {
+      backgroundImage: `url('${backgroundUrl}')`,
+      backgroundColor,
+    };
+
+    const classes = classnames('countdownTimer', {
+      [`is-${textColor}`]: !!textColor,
+    });
+
+    return (
+      <Fragment>
+        <InspectorControls>
+          <PanelBody title={ __('Options', 'benenson') }>
+            <SelectControl
+              label={ __('Text color', 'benenson') }
+              value={ textColor }
+              onChange={ color => setAttributes({ textColor: color }) }
+              options={ colourOptions }
+            />
+            <DateTimePicker
+              currentDate={ date }
+              onChange={ newDate => setAttributes({ date: newDate }) }
+              // is12Hour={ is12HourTime }
+            />
+          </PanelBody>
+          <PanelBody title={ __('Background color', 'benenson') }>
+            <ColorPicker
+              color={ backgroundColor }
+              onChangeComplete={ color => setAttributes({ backgroundColor: color.hex }) }
+              disableAlpha
```

Comment:
If this is a boolean prop, it might be worth making it explicit (`disableAlpha={ true }`), so that there's no confusion.